repos/system_upgrade/common/files/upgrade_paths.json

```diff
@@ -5,15 +5,19 @@
         "8.8": ["9.2"],
         "8.9": ["9.3"],
         "8.10": ["9.4"],
+        "9.6": ["10.0"],
         "7": ["8.6", "8.8", "8.9", "8.10"],
-        "8": ["9.0", "9.2", "9.3", "9.4"]
+        "8": ["9.0", "9.2", "9.3", "9.4"],
+        "9": ["10.0"]
```

Comment:
I'd say for each new upgrade path we should add a packit test. And ideally this should be done in this commit.

Reply:
you are right. Q is whether it will need some additional tweaks regarding major upgrade path change (sometimes it could include some specific infra changes). we should check that later. but with minor update paths we should start doing that definitely in this way.